### PR TITLE
Add benchmarks

### PR DESCRIPTION
**Pull request by Cheshiriks**
Add benchmarks

### CODE REVIEW

File: benchmarks/spec.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    # license used in the project from the list provided by this repo
+    license = 'Copyright 1995 - 2022 Standard Performance Evaluation Corporation'
+    # number of tests/cases/scenarios/benchmarks (should not be 0)
+    scenarios_num = 4
+    # free text with a small desciption of the benchmark
+    description = 'The benchmarks aim to test "real-life" situations. There are several benchmarks testing Java scenarios, from simple computation (SPECjbb) to a full system with Java EE, database, disk, and network (SPECjEnterprise).'
+
+[links]
+    # the link to the home page of the benchmark
+    homepage = 'https://www.spec.org/benchmarks.html'
```
Comment (orchestr7) [inline benchmarks/spec.toml]: please add C/C++: https://www.spec.org/cpu2017/

Comment (Cheshiriks) [reply benchmarks/spec.toml]: Added

File: benchmarks/kotlinx_benchmark.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[general]
+    # benchmark name
+    name = 'kotlinx'
```
Comment (petertrr) [inline benchmarks/kotlinx_benchmark.toml:3]: Why do we need this here? It's a library for benchmarks, not a test suite for benchmarking

Comment (orchestr7) [reply benchmarks/kotlinx_benchmark.toml:3]: we will have a category `tools` also (frameworks/libs/etc)

**Review by orchestr7 — APPROVED** [review]

**Pull request merged**